### PR TITLE
feat: resize image for TinEye search

### PR DESCRIPTION
**Pull request by dimatretyak**
This PR resolves []

### Some advice about image for TinEye
* Mobile images should be 600 pixels in the smallest dimension
* Recommended average image size is 100 kB
* Use a quality setting of 85% or higher for JPEG images
* Uploaded images must be JPEG, PNG, WebP, GIF, BMP or TIFF format

#### Image 1
```javascript
// Before resize
{
  "width": 900,
  "height": 900,
  "size": 5083 // ~ 5kb
}

// After resize
{
  "width": 600,
  "height": 600,
  "size": 3003 // ~ 3kb
}
```

#### Image 2
```javascript
// Before resize
{
  "width": 1600,
  "height": 1200,
  "size": 281044 // ~281kb
}

// After resize
{
  "width": 800,
  "height": 600,
  "size": 72232 // ~72kb
}
```

#### Image 3
```javascript
// Before resize
{
  "width": 1080,
  "height": 2160,
  "size": 1184278 // ~1mb
}

// After resize
{
  "width": 600,
  "height": 1200,
  "size": 70170 // ~70kb
}
```

#### Image 4
```javascript
// Before resize
{
  "width": 1440,
  "height": 900,
  "size": 7109 // ~7kb
}

// After resize
{
  "width": 960,
  "height": 600,
  "size": 4262 // ~4kb
}
```

#### Image 5
```javascript
// Before resize
{
  "width": 1280,
  "height": 962,
  "size": 73096 // ~73kb
}

// After resize
{
  "width": 798,
  "height": 600,
  "size": 35141 // ~35kb
}
```

#### Image 6
```javascript
// Before resize
{
  "width": 4000,
  "height": 3000,
  "size": 4206311 // ~4mb
}

// After resize
{
  "width": 450,
  "height": 600,
  "size": 43879 // ~43kb
}
```

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [x] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- resize image for TinEye search - dimatretyak

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: /docs/adding_state_migrations.md
[feature flag]: /docs/developing_a_feature.md
[docs]: /docs/README.md

### CODE REVIEW

File: src/app/Scenes/Fair/Fair.tsx [hunk 1]
```diff
@@ -176,18 +177,34 @@ export const Fair: React.FC<FairProps> = ({ fair }) => {
...
+        resizedWidth = 600
+      }
+
+      const resizedImage = await resizeImage({
         uri: image.path,
-        name: getFileNameByPath(image.path),
-        type: image.mime,
+        width: resizedWidth,
+        height: resizedHeight,
+        quality: 80,
```
Comment (gkartalis) [inline src/app/Scenes/Fair/Fair.tsx:201]: > Use a quality setting of 85% or higher for JPEG images

**question:** Not sure if these two are the same thing but shouldn't this be `85%`?

**Review by gkartalis — APPROVED** [review]
Looking great! 🚀

**Comment by gkartalis** [comment]
Also thanks for adding the examples 👍

File: src/app/Scenes/Fair/Fair.tsx (hunk 1 above)
Comment (dimatretyak) [reply src/app/Scenes/Fair/Fair.tsx:201]: Fixed

**Pull request merged**